### PR TITLE
Implements POST access/logout endpoint

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: src/ts/auth/AuthAPI.ts [hunk 1]
```diff
@@ -47,6 +48,18 @@ export default class AuthAPI {
     });
   }
 
+  public postLogout(): AxiosPromise {
+    const config: AxiosRequestConfig = {
+      withCredentials: true,
+      method: 'post',
+      url: `${AuthAPI.URL.ACCESS}/${AuthAPI.URL.LOGOUT}`
```
Comment (lipis) [inline src/ts/auth/AuthAPI.ts]: natural order and comma

Comment (Yserz) [inline src/ts/auth/AuthAPI.ts]: I refer to https://github.com/wireapp/wire-web-api-client/pull/14

Comment (lipis) [inline src/ts/auth/AuthAPI.ts]: damn :|

Comment (lipis) [inline src/ts/auth/AuthAPI.ts]: rmgvv5r

**Pull request merged**